try {
    hello()
} catch (error) {
    console.log("Error");
}